x-pack/test/detection_engine_api_integration/security_and_spaces/group1/get_rule_execution_results.ts

```diff
@@ -122,7 +122,7 @@ export default ({ getService }: FtrProviderContext) => {
       expect(response.body.events[0].security_status).to.eql('partial failure');
       expect(
         response.body.events[0].security_message.startsWith(
-          'Check privileges failed to execute ResponseError: index_not_found_exception: [index_not_found_exception] Reason: no such index [no-name-index]'
+          'This rule is attempting to query data from Elasticsearch indices listed in the "Index pattern" section of the rule definition, however no index matching: ["no-name-index"] was found.'
```

Comment:
Does it work with Data Views as well? Just curious if, firstly it is possible to have non-existing index within Data View and secondly, if it is possible then whether we need to adjust the messaging saying `... listed in the "Index pattern" ...`

Reply:
Yeah that's a good point, we should update the copy here to account for data views or raw index patterns. It's possible that the data view could reference no existing indices. The logic works fine in that case, it's just the error message that doesn't quite line up.